include/seqan3/io/stream/iterator.hpp

```diff
@@ -381,42 +377,3 @@ class fast_ostreambuf_iterator
 };
 
 } // namespace seqan3::detail
-
-namespace seqan3
-{
-
-/*!\addtogroup stream
- * \{
- */
-
-#ifdef __cpp_lib_ranges
-
-/*!\typedef seqan3::ostream_iterator
-* \brief Alias for std::ostream_iterator. Writes successive elements onto the output stream from which it was
-*        constructed.
-*/
-using SEQAN3_DOXYGEN_ONLY(ostream_iterator =) ::std::ostream_iterator;
-
-/*!\typedef seqan3::ostreambuf_iterator
- * \brief Alias for std::ostreambuf_iterator. Writes successive characters onto the output stream from which it
- *        was constructed.
- */
-using SEQAN3_DOXYGEN_ONLY(ostreambuf_iterator =) ::std::ostreambuf_iterator;
-
-#else
-
-/*!\typedef seqan3::ostream_iterator
- * \brief Alias for ranges::ostream_iterator. Writes successive elements onto the output stream from which it was
- *        constructed.
- */
-using SEQAN3_DOXYGEN_ONLY(ostream_iterator =) ::ranges::ostream_iterator;
-
-/*!\typedef seqan3::ostreambuf_iterator
- * \brief Alias for ranges::ostreambuf_iterator. Writes successive characters onto the output stream from which it
- *        was constructed.
- */
-using SEQAN3_DOXYGEN_ONLY(ostreambuf_iterator =) ::ranges::ostreambuf_iterator;
-
```

Comment:
> Just one question to be sure. The renamed iterator types have nothing to do with our faster stream iterator implementations?

As you can see, this has nothing to do with our faster stream iterators. This is from way back when we didn't have the `seqan3/std` folder for "standard" implementations.

That means `seqan3::ostreambuf_iterator` was either `ranges::ostreambuf_iterator` (pre C++20) or `std::ostreambuf_iterator` (C++20).

Reply:
Ok, thanks for clarifying.